packages/theme/getters/productGetters.ts

```diff
@@ -199,15 +198,37 @@ export const getFormattedPrice = (price: number) => {
   }).format(price);
 };
 
-export const getBreadcrumbs = (product: ProductInterface, category?: Category): AgnosticBreadcrumb[] => {
+const getCategoryBreadcrumbs = (category: CategoryInterface): AgnosticBreadcrumb[] => {
+  let breadcrumbs = [];
+
+  if (!category) {
+    return [];
+  }
+
+  if (Array.isArray(category?.breadcrumbs)) {
+    breadcrumbs = category.breadcrumbs.map((breadcrumb) => ({
+      text: breadcrumb.category_name,
+      link: `/c/${breadcrumb.category_url_path}${category.url_suffix || ''}`,
+    } as AgnosticBreadcrumb));
+  }
+
+  breadcrumbs.push({
+    text: category.name,
+    link: `/c/${category.url_path}${category.url_suffix || ''}`,
+  } as AgnosticBreadcrumb);
+
+  return breadcrumbs;
```

Comment:
```suggestion
  const breadcrumbs = Array.isArray(category?.breadcrumbs)) 
    ? category.breadcrumbs.map((breadcrumb) => ({
      text: breadcrumb.category_name,
      link: `/c/${breadcrumb.category_url_path}${category.url_suffix || ''}`,
    })
    : [];
    
  return [... breadcrumbs, {
    text: category.name,
    link: `/c/${category.url_path}${category.url_suffix || ''}`,
  }]
```
optional: just an idea i had for this, this gets rid of the let breadcrumbs = [] mutation